ql/src/java/org/apache/hadoop/hive/ql/security/authorization/HiveCustomStorageHandlerUtils.java

```diff
@@ -34,6 +34,8 @@ public class HiveCustomStorageHandlerUtils {
 
     public static final String WRITE_OPERATION_IS_SORTED = "file.sink.write.operation.sorted.";
 
+    public static final String WRITE_OPERATION_FOR_MERGE_TASK = "file.sink.write.operation.merge.";
```

Comment:
is this just a marker that we are inside of merge task context? if yes, maybe just
`file.sink.merge.task`/enabled

Reply:
Done.